tests/test_tidyversetopandas.py

```diff
@@ -1,23 +1,100 @@
 from tidyversetopandas import tidyversetopandas
 import pandas as pd
+import pytest
+<<<<<<< HEAD
+=======
 
+>>>>>>> main
 
-def test_mutate_no_func():
-    """Test mutate function without a function"""
+@pytest.fixture
+def input_df_1():
     df = pd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
-    df = tidyversetopandas.mutate(df, new_b=7)
+    return df
+
+
+@pytest.fixture
+def input_df_2():
+    df = pd.DataFrame({"A": [1, 2, 3], "B": [4, 5, 6], "C": ["a", "b", "c"]})
+    return df
+
+
+def test_mutate_no_func(input_df_1):
+    """Test mutate function without a function"""
+    df = tidyversetopandas.mutate(input_df_1, new_b=7)
     assert df["new_b"].tolist() == [7, 7, 7]
 
 
-def test_mutate_same_col():
+def test_mutate_same_col(input_df_1):
     """Test mutate function that overwrites existing column"""
-    df = pd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
-    df = tidyversetopandas.mutate(df, b=lambda x: x["b"] * 2)
+    df = tidyversetopandas.mutate(input_df_1, b=lambda x: x["b"] * 2)
     assert df["b"].tolist() == [8, 10, 12]
 
 
-def test_mutate_new_col():
+def test_mutate_new_col(input_df_1):
     """Test mutate function that creates new column"""
-    df = pd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
-    df = tidyversetopandas.mutate(df, c=lambda x: x["a"] + x["b"])
+    df = tidyversetopandas.mutate(input_df_1, c=lambda x: x["a"] + x["b"])
     assert df["c"].tolist() == [5, 7, 9]
+
+
+<<<<<<< HEAD
+def test_select_single_column():
+    """Test select function with a single column"""
+    df = pd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
```

Comment:
you can use fixture method for this
```suggestion
    input_df_1
```